jlm/util/file.hpp

```diff
@@ -18,15 +18,15 @@ namespace jlm::util
 class filepath final
 {
 public:
-  inline filepath(const std::string & path)
+  filepath(const std::string & path)
```

Comment:
Since you touch it, it should have explicit.

Reply:
Many places use it implicitly, so we agreed to not add explicit